packages/frameworks/src/frameworks.ts

```diff
@@ -1523,6 +1524,7 @@ export const frameworks = [
     devCommand: 'zola serve --port $PORT',
     buildCommand: 'zola build',
     getOutputDirName: async () => 'public',
+    defaultVersion: 'v0.13.0',
```

Comment:
Let's drop the `v` prefix since we can add it in the docs if we want but its hard to remove it.

Also the env var doesn't accept the `v` prefix.